src/Sulu/Bundle/PreviewBundle/Resources/js/containers/Preview/Preview.js

```diff
@@ -137,9 +137,9 @@ class Preview extends React.Component<Props> {
         );
 
         this.typeDisposer = reaction(
-            () => toJS(formStore.type),
-            (type) => {
-                previewStore.updateContext(type, formStore.data).then(this.setContent);
+            () => toJS(formStore.schema),
```

Comment:
i guess this needs to be changed to update the context only after the new schema is loaded, right?

Reply:
yes otherwise the preview is reloaded to early